basic_system/src/system_implementation/system/io_subsystem.rs

```diff
@@ -474,6 +476,7 @@ where
                 .oracle
                 .create_oracle_access_iterator::<InitializeIOImplementerIterator>(())
                 .unwrap();
+            // TODO(refactoring): read only state commitment
```

Comment:
Can you link to a linear issue? Just opened https://github.com/matter-labs/zksync-os/pull/52 to do it